Makefile

```diff
@@ -87,6 +87,8 @@ hiveadmission:
 # Run against the configured Kubernetes cluster in ~/.kube/config
 .PHONY: run
 run: generate fmt vet
+	oc scale -n hive deployment.v1.apps/hive-controllers --replicas=0
```

Comment:
Why did the makefile need such change? How does this work if I do not have a OCP cluster installed when run makefile?

Also if only scale `hive-controllers` to 0, does the `hive-operator` will create new instance for hive controllers?

Reply:
Normal path to install is make deploy, it *might* be possible to just make install the crds and then make run but I don't think we document that path anywhere. In general it should be safe to assume hive-controllers is there.

I don't *think* the hive-operator will fight for scale but I'm not positive, if it reconciled a HiveConfig it might flip it back.

I'm going to remove these changes for now until I'm more sure they should be there.